CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/prometheus_server_role/tree/develop)
+### Added
+- *[#37](https://github.com/idealista/prometheus_server_role/issues/37) Replace copy with template with custom delimiters for rules* @caldito
 
```

Comment:
In addition to the message in "added" section we should state in a "changed" section that the default delimiters have been changed.

Reply:
Actually there is not a change in that because there were no templates prior to this, just a simple copy.